isic/zip_download/tests/test_zip_download.py

```diff
@@ -23,43 +22,40 @@ def random_images_with_licenses(image_factory):
 
 
 @pytest.mark.django_db
-def test_zip_download_licenses(authenticated_api_client, random_images_with_licenses):
-    r = authenticated_api_client.post(reverse("zip-download/api/url"), {"query": ""})
-    assert r.status_code == 200, r.data
-    parsed_url = urlparse(r.data)
+def test_zip_download_licenses(authenticated_client, random_images_with_licenses):
+    r = authenticated_client.post(
+        "/api/v2/zip-download/url/", {"query": ""}, content_type="application/json"
+    )
+    assert r.status_code == 200, r.json()
+    parsed_url = urlparse(r.json())
     token = parse_qs(parsed_url.query)["zsid"]
 
-    r = authenticated_api_client.get(
-        reverse("zip-download/api/file-listing"), data={"token": token[0]}
-    )
+    r = authenticated_client.get("/api/v2/zip-download/file-listing/", data={"token": token[0]})
     assert r.status_code == 200, r.json()
 
-    assert any(["CC-0" in result["url"] for result in r.json()["results"]])
-    assert any(["CC-BY" in result["url"] for result in r.json()["results"]])
-    assert not any(["CC-BY-NC" in result["url"] for result in r.json()["results"]])
+    assert any("CC-0" in result["url"] for result in r.json()["results"])
```

Comment:
Out of curiosity, did you notice these yourself or did any tool point them out to you?

Reply:
Just noticed myself.